### PR TITLE
feat(api): add model for storing clubs information

### PR DESCRIPTION
**Pull request by PranshuNayak**

### CODE REVIEW

File: api/api/clubs/apps.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class ClubsConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'api_clubs'
```
Comment (Samy-33) [inline api/api/clubs/apps.py:6]: name should be `api.clubs`
Look at the old apps for similarity.
For example: https://github.com/BitByte-TPC/gymkhana/blob/master/api/api/accounts/apps.py

File: api/api/clubs/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class Club (models.Model):
+    CULTURAL_CLUB = 'Cultural'
+    SCIENCE_AND_TECH_CLUB = 'Science & Technology'
+    SPORTS_CLUB = 'Sports'
+
+    CLUB_TYPE_CHOICES = [
+        ('Cultural', CULTURAL_CLUB),
+        ('S&T', SCIENCE_AND_TECH_CLUB),
+        ('Sports', SPORTS_CLUB)
```
Comment (Samy-33) [inline api/api/clubs/models.py:12]: Is there an upside to keeping these as variables? `CULTURAL_CLUB`, `SCIENCE_AND...` etc.

**Review by Samy-33 — CHANGES_REQUESTED** [review]

File: api/api/clubs/models.py (hunk 2 above)
Comment (PranshuNayak) [reply api/api/clubs/models.py:12]: I suppose use of variables increases code readability and provides single point of change if necessary.

Comment (Samy-33) [reply api/api/clubs/models.py:12]: For readability: CULTURAL_CLUB as a string is also pretty readable

But yes, single point of change in variables is useful.

Comment (PranshuNayak) [reply api/api/clubs/models.py:12]: So should I change or keep the variables .

**Review by Samy-33 — APPROVED** [review]

**Pull request merged**